spring-cloud-sleuth-core/pom.xml

```diff
@@ -286,20 +289,28 @@
 			<artifactId>brave-instrumentation-grpc</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>io.zipkin.reporter2</groupId>
+			<artifactId>zipkin-reporter-metrics-micrometer</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>io.micrometer</groupId>
```

Comment:
@shakuzen reminds me.. we probably should have had this set to "provided" scope in upstream anyway